cookbooks/aws-parallelcluster-slurm/recipes/install_slurm.rb

```diff
@@ -19,6 +19,20 @@
   action :purge
 end
 
+apt_update if node['platform'] == 'ubuntu'
```

Comment:
why do we need this?

Reply:
apt_update is needed for ubuntu to install `libhttp-parser-del`,  otherwise the packages can not be found.
Instruction here also mentioned `sudo apt-get update`
https://ubuntu.pkgs.org/18.04/ubuntu-main-amd64/libhttp-parser-dev_2.7.1-2_amd64.deb.html

Reply:
don't we already run apt_update in base.rb?

Reply:
Was testing with running single recipe and it was required.  Removed it from the recipe since it was run before in other recipes